frontend.go

```diff
@@ -16,9 +16,14 @@ package casbin
 
 import (
 	"encoding/json"
+	"errors"
 )
 
-func CasbinJsGetPermissionForUser(e *Enforcer, user string) ([]byte, error) {
+func CasbinJsGetPermissionForUser(e IEnforcer, user string) ([]byte, error) {
+	_, ok := e.(IEnforcer)
+	if !ok {
+		return nil, errors.New("Unsupported type")
+	}
```

Comment:
I believe this is dead code, if `e` is not of type `IEnforcer` the code won't compile. AFAICT you can just remove this check.

Reply:
You are right.